recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "v0.5.1" %}
+{% set version = "v0.5.2" %}
 
 package:
   name: nbgrader
   version: {{ version }}
 
 source:
   url: https://github.com/jupyter/nbgrader/archive/{{ version }}.tar.gz
```

Comment:
I moved the older versions with the `v` string to a "broken" label in case people need them. So the previous versions are still installable with the command:

`conda install nbgrader -c conda-forge/label/broken`

But that label won't interfere with the new releases from now on.